.rubocop.yml

```diff
@@ -17,6 +17,15 @@ Style/ClassAndModuleChildren:
 Style/Encoding:
   Enabled: false
 
+RSpec/MessageSpies:
+  Exclude:
+    - 'spec/controllers/users/omniauth_callbacks_controller_spec.rb'
+
+Metrics/AbcSize:
+  Exclude:
+    - 'app/interactors/omniauth_facebook.rb'
```

Comment:
Do not exclude specific interactors or controllers. Try to refactor your code or use `# rubocop:disable ...`